torch/_inductor/ir.py

```diff
@@ -5246,6 +5249,9 @@ def handle_aliasing_and_mutation(info, arg):
                 return
             # can_auto_functionalize already filters out mutable List[Tensor].
             # We can support this in the future, but this is very uncommon.
+            assert (
+                not is_list_tensor
+            ), "Operators that accept mutable List[Tensor] are not yet supported in inductor"
```

Comment:
Can you add a test that shows that this is the error one gets if they try backend="inductor"?